srtcore/sync.h

```diff
@@ -75,13 +77,14 @@ class Duration
     inline bool operator<(const Duration& rhs) const { return m_duration < rhs.m_duration; }
 
 public: // Assignment operators
-    inline void operator*=(const double mult) { m_duration = static_cast<int64_t>(m_duration * mult); }
+    inline void operator*=(const int64_t mult) { m_duration = static_cast<int64_t>(m_duration * mult); }
```

Comment:
Why is this required?

Reply:
This was changed for compatibility with C++11 imlementation, that does not define multiplications by float point types.